### DISCUSSION TITLE
Error returned by the simplification algorithm

### DISCUSSION THREAD

**Discussion by GiuseppeCassisi96** [zeux/meshoptimizer#735] [Q&A] [answered]
Hey
The error returned by `meshopt_simplify()` corresponds to the Hausdorff distance between the original mesh and the simplified one ?

Many thanks

**Comment by zeux** [comment, marked as answer]
It's reasonable to think about the error as an approximation of Hausdorff metric (accounting for normalization, e.g. if ErrorAbsolute flag is not specified then you'd need to multiply by mesh size as returned by meshopt_simplifyScale). Note that it's an approximation - it's not computed exactly as Hausdorff metric, and in some cases the values are definitely significantly different. But directionally this is correct, simplification of a planar patch that doesn't move the edge vertices will yield error of zero as even though individual vertices move significantly, they move on the mesh's surface; if an edge vertex does move, the error will be proportional to the moved distance because the edge of the patch deformed.
